app-laptop/thinkfan/thinkfan-0.9.3.ebuild

```diff
@@ -18,17 +18,21 @@ IUSE="atasmart"
 DEPEND="atasmart? ( dev-libs/libatasmart )"
 RDEPEND="${DEPEND}"
 
-S="${WORKDIR}"
-
 src_prepare() {
+	default
```

Comment:
if you inherit cmake-utils, **never** call `default` but instead `cmake-utils_src_prepare`